CMakeLists.txt

```
@@ -8,32 +8,38 @@ function(packageProject)
   include(GNUInstallDirs)
 
   cmake_parse_arguments(
-    PROJECT "NO_VERSION_SUFFIX"
-    "NAME;VERSION;INCLUDE_DIR;INCLUDE_DESTINATION;BINARY_DIR;COMPATIBILITY;VERSION_HEADER;NAMESPACE"
-    "DEPENDENCIES" ${ARGN}
+    PROJECT
+    ""
+    "NAME;VERSION;INCLUDE_DIR;INCLUDE_DESTINATION;BINARY_DIR;COMPATIBILITY;VERSION_HEADER;NAMESPACE;DISABLE_VERSION_SUFFIX"
+    "DEPENDENCIES"
+    ${ARGN}
   )
 
-  if(DEFINED PROJECT_NO_VERSION_SUFFIX)
+  # optional feature: TRUE or FALSE or UNDEFINED! These variables will then hold the respective
+  # value from the argument list or be undefined if the associated one_value_keyword could not be
+  # found.
+  if(DEFINED PROJECT_DISABLE_VERSION_SUFFIX AND PROJECT_DISABLE_VERSION_SUFFIX)
```

If a value is truthy, is also has to be defined.

```suggestion
  if(PROJECT_DISABLE_VERSION_SUFFIX)
```

---

I've gone ahead and updated the condition.